var/spack/repos/builtin/packages/camellia/package.py

```diff
@@ -0,0 +1,59 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Camellia(CMakePackage):
+    """Camellia: user-friendly MPI-parallel adaptive finite element package,
+       with support for DPG and other hybrid methods, built atop Trilinos.
+    """
+
+    homepage = "https://bitbucket.org/nateroberts/Camellia"
+    url      = "https://bitbucket.org/nateroberts/camellia.git"
+
+    maintainers = ['nvrober-sandia']
+
+    version('master',
+            git='https://bitbucket.org/nateroberts/camellia.git', branch='master')
+
+    depends_on('trilinos~shared+amesos+amesos2+anasazi+belos+epetra+epetraext+exodus+ifpack+ifpack2+intrepid+intrepid2+kokkos+ml+muelu+sacado+shards+teuchos+tpetra+zoltan+mumps+superlu-dist+hdf5+zlib+pnetcdf@master,12.12.1:')
+    depends_on('moab-metis-parmetis', when='+moab')
+
+    def cmake_args(self):
+        spec = self.spec
+        options = []
+
+        options.extend([
```

Comment:
Instead of creating an empty list and extending it, why not create a populated list?

Reply:
Fixed.  This was simply an artifact of copying and pasting from the nalu ```package.py```.  Thanks, @adamjstewart, for the review!  I'll push the fixes once my test install completes.